lib/dndmonster.sty

```diff
@@ -357,44 +357,48 @@
     
     \emph {#1}
   }
-\ExplSyntaxOff
 
-\newcommand\crexp[1]{%
-	\IfStrEqCase{#1}{%
-		{0}{#1 (0 \xpname)}%
-		{1/10}{0 (10 \xpname)}%
-		{1/8}{#1 (25 \xpname)}%
-		{1/4}{#1 (50 \xpname)}%
-		{1/2}{#1 (100 \xpname)}%
-		{1}{#1 (200 \xpname)}%
-		{2}{#1 (450 \xpname)}%
-		{3}{#1 (700 \xpname)}%
-		{4}{#1 (\numprint{1100} \xpname)}%
-		{5}{#1 (\numprint{1800} \xpname)}%
-		{6}{#1 (\numprint{2300} \xpname)}%
-		{7}{#1 (\numprint{2900} \xpname)}%
-		{8}{#1 (\numprint{3900} \xpname)}%
-		{9}{#1 (\numprint{5000} \xpname)}%
-		{10}{#1 (\numprint{5900} \xpname)}%
-		{11}{#1 (\numprint{7200} \xpname)}%
-		{12}{#1 (\numprint{8400} \xpname)}%
-		{13}{#1 (\numprint{10000} \xpname)}%
-		{14}{#1 (\numprint{11500} \xpname)}%
-		{15}{#1 (\numprint{13000} \xpname)}%
-		{16}{#1 (\numprint{15000} \xpname)}%
-		{17}{#1 (\numprint{18000} \xpname)}%
-		{18}{#1 (\numprint{20000} \xpname)}%
-		{19}{#1 (\numprint{22000} \xpname)}%
-		{20}{#1 (\numprint{25000} \xpname)}%
-		{21}{#1 (\numprint{33000} \xpname)}%
-		{22}{#1 (\numprint{41000} \xpname)}%
-		{23}{#1 (\numprint{50000} \xpname)}%
-		{24}{#1 (\numprint{62000} \xpname)}%
-		{25}{#1 (\numprint{75000} \xpname)}%
-		{26}{#1 (\numprint{90000} \xpname)}%
-		{27}{#1 (\numprint{105000} \xpname)}%
-		{28}{#1 (\numprint{120000} \xpname)}%
-		{29}{#1 (\numprint{135000} \xpname)}%
-		{30}{#1 (\numprint{155000} \xpname)}%
-	}[#1]%
-}
+\NewDocumentCommand {\DndCRExp} { m }
```

Comment:
We need to deprecate public commands (not containing `@`) before removing them.

It's unlikely that anyone used `\crexp` directly, but let's make it an alias of the new command use the the `\dnd@deprecate` helper here.

Reply:
`\crexp` was an internal function before I knew how to write internal functions. When this class goes full LaTeX3, it will be an internal function. So I will change the name of `\DndCRExp` to `\dnd@crexp`.

Since this was never part of a release (0.6.0 was tagged before this came out), does it need to be deprecated?

Reply:
> Since this was never part of a release (0.6.0 was tagged before this came out), does it need to be deprecated?

Ah, my mistake. I thought it went out with 0.6.0.

No, in that case we don't need to create an alias.